MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control to handle the data.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following attribute to handle the data, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name] `Link`
```

Comment:
include links for all display attributes parameters

https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.name?view=net-7.0#system-componentmodel-dataannotations-displayattribute-name

Reply:
links are added